dg/media/dashboard.html

```diff
@@ -19,7 +19,7 @@
     <LINK REL="stylesheet"  HREF="/media/coco/app/styles/css/tabletools_custom.css">
     <LINK rel="icon" type="image/x-icon" href="/media/coco/app/images/favicon.ico">
 
-    <script data-main="/media/coco/dist/scripts/main" src="/media/coco/app/scripts/libs/require.js"></script>
+    <script data-main="/media/coco/app/scripts/main" src="/media/coco/app/scripts/libs/require.js"></script>
```

Comment:
why is directory changed from dist to app

Reply:
Explained to you.